index.js

```diff
@@ -0,0 +1,29 @@
+require("dotenv").config();
+const express = require("express");
+const cors = require("cors");
+const { connectToDatabase } = require("./db/dbConnection");
+const {
+  productsRouter,
+  cartsRouter,
+  usersRouter,
+  wishlistsRouter,
+} = require("./routes/index");
+
+const PORT = process.env.PORT || 3000;
+
+connectToDatabase();
+
+const app = express();
+app.use(cors());
+app.use(express.json());
+
+app.use("/products", productsRouter);
+app.use("/cart", cartsRouter);
+app.use("/user", usersRouter);
+app.use("/wishlist", wishlistsRouter);
+
+app.get("/", (req, res) =>
+  res.send("API for Jarvis Store ecommerce application.")
+);
+
```

Comment:
Add a global route & error handler here so that any unspecified route can be handled and similarly errors occurring inside routes will be handled too.